src/components/YourCompany/NameTranslations/ListNameTranslation.vue

```diff
@@ -0,0 +1,210 @@
+                    class="actions__more-actions__btn"
+                    :disabled="isAddingNameTranslation || translation.action === actionTypes.REMOVED">
+                    <v-icon>mdi-menu-down</v-icon>
+                  </v-btn>
+                </template>
+                <v-list class="actions__more-actions">
+                  <v-list-item  @click="emitNameEdit(index)">
+                    <v-list-item-subtitle>
+                      <v-icon small>mdi-pencil</v-icon>
+                      <span class="ml-1">Edit</span>
+                    </v-list-item-subtitle>
+                  </v-list-item>
+                  <v-list-item  @click="emitRemoveName(index)">
+                    <v-list-item-subtitle>
+                      <v-icon small>mdi-delete</v-icon>
+                      <span class="ml-1">Remove</span>
+                    </v-list-item-subtitle>
+                  </v-list-item>
+                </v-list>
+              </v-menu>
+            </span>
+          </div>
+        </v-col>
+        <v-col v-else>
+          <div class="actions">
+            <span class="edit-action">
+              <v-btn
+                small
+                text
+                color="primary"
+                :disabled="isAddingNameTranslation"
+                @click="emitNameEdit(index)">
+                  <v-icon small>mdi-pencil</v-icon>
+                  <span>Edit</span>
+              </v-btn>
+            </span>
+            <!-- more actions menu -->
+            <span>
+              <v-menu offset-y>
+                <template v-slot:activator="{ on }">
+                  <v-btn
+                    text
+                    small
+                    v-on="on"
+                    color="primary"
+                    class="actions__more-actions__btn"
+                    :disabled="isAddingNameTranslation">
+                    <v-icon>mdi-menu-down</v-icon>
+                  </v-btn>
+                </template>
+                <v-list class="actions__more-actions">
+                  <v-list-item  @click="emitRemoveName(index)">
+                    <v-list-item-subtitle>
+                      <v-icon small>mdi-delete</v-icon>
+                      <span class="ml-1">Remove</span>
+                    </v-list-item-subtitle>
+                  </v-list-item>
+                </v-list>
+              </v-menu>
+            </span>
+          </div>
+        </v-col>
+      </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Prop, Vue, Emit } from 'vue-property-decorator'
+
+// Interfaces
+import { NameTranslationDraftIF } from '@/interfaces'
+
+// Enums
+import { ActionTypes } from '@/enums'
+
+@Component({})
+export default class ListNameTranslation extends Vue {
+  @Prop({ default: () => [] })
+  private translationList: NameTranslationDraftIF[]
+
+  @Prop({ default: false })
+  private isAddingNameTranslation: boolean
+
+  private actionTypes = ActionTypes
```

Comment:
I recommend you capitalize the left side, as it's still a "type". It just needs to be in the script so the Vue template can see it.